### PR TITLE
Version 1.4.0 of plugin

### PR DESCRIPTION
**Pull request by growdev**
This version adds support for v3 of the ConvertKit API. #10

### CODE REVIEW

File: lib/convertkit-api.php [hunk 1]
```diff
@@ -5,183 +5,223 @@
...
+	}
+
+	/**
+	 * Remove subscription from a form
+	 *
+	 * @param string $form_id Resource ID
+	 * @param array $options Array of user data
+	 * @return object Response object
+	 */
+	public function form_unsubscribe($form_id, $options) {
```
Comment (davidlamarwheeler) [inline lib/convertkit-api.php]: Unfortunately, API v3 doesn't support this one

Comment (growdev) [inline lib/convertkit-api.php]: Okay, I'll pull it from the plugin.

**Comment by growdev** [comment]
@davidlamarwheeler Please review 3 additional commits and let me know if this is good to release.

**Comment by davidlamarwheeler** [comment]
Looks solid, sir. :shipit:

**Pull request merged**